db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_suspend_vm_ok.yaml

```diff
@@ -0,0 +1,16 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name:
+    name: USER_SUSPEND_VM_OK
```

Comment:
@gmcculloug The only RHEVM events during suspending a VM are USER_SUSPEND_VM when it is initiated and USER_SUSPEND_VM_OK when it is suspended. 

Don't know when USER_SUSPEND_VM_FINISH_SUCCESS is sent. 

I tried but could not find any information about the difference between these two events other than their event codes.

VM refresh would update all the states of the VM but it may take a while. So @Fryguy recommended set the suspended state right away as it is an important information and other actions may depend upon it.